lib/matplotlib/axes/_base.py

```diff
@@ -1287,25 +1287,59 @@ def apply_aspect(self, position=None):
 
     def axis(self, *v, **kwargs):
         """
-        Convenience method for manipulating the x and y view limits
-        and the aspect ratio of the plot. For details, see
-        :func:`~matplotlib.pyplot.axis`.
+        Call signature::
+
```

Comment:
No problem, I wasn't aware you're encouraging this format, and it's not clear from the current docstrings. Before I commit, is this ok? Seeing as the function only takes in optional arguments, I wasn't sure if to include a 'Parameters' section or to put both `*v` and `**kwargs` in 'Other parameters' 
```
"""Set axis properties

        Convenience method for manipulating axis properties such as
        the x and y view limits and the aspect ratio of the plot.

        For details, see
        :func:`~matplotlib.pyplot.axis`.

        Parameters
        ----------
        *v
            Axis toggle, aspect ratio and limits properties.

        Other parameters
        ----------------
        **kwargs
            Axis limit properties

        *kwargs* are passed on to :meth:`set_xlim` and
        :meth:`set_ylim`

        Returns
        -------
        v : list of axis limits ``(xmin, xmax, ymin, ymax)``.

        """
```